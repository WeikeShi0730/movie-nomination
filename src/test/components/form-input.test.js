import React from "react";
import { shallow } from "enzyme";
import FormInput from "../../components/form-input/form-input.component";

it("should render FormInput component", () => {
  expect(shallow(<FormInput />).getElement()).toMatchSnapshot();
});
