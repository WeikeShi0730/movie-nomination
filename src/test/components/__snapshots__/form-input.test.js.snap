// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FormInput component 1`] = `
<div
  className="group"
>
  <input
    className="form-input"
  />
</div>
`;
